### DISCUSSION TITLE
Storage path on Photoprism under Umbrel

### DISCUSSION THREAD

**Discussion by giancagentile** [photoprism/photoprism#3836] [Q&A] [answered]
Hi. I've just installed PhptoPrism under Umbrel and I'd like to change the storage path to a different disk other than the umbrel disk. How can it be done?

Thanks!

**Comment by graciousgrey** [comment, marked as answer]
When using docker-compose, volumes can be defined as described here:

https://docs.photoprism.app/getting-started/docker-compose/#volumes

Does Umbel provide configuration options for the originals, import and storage volumes in its user interface? 
If not maybe another Umbrel user can help here.

**Reply by giancagentile** [reply]
Thanks for the answer. Umbre UI for PfotoPrism does not show any config options.

**Reply by lastzero** [reply]
You can alternatively use an options.yml file if you know (or can ask their community) where the storage/config folder is located on your Umbrel device:

- https://docs.photoprism.app/getting-started/config-files/

**Reply by giancagentile** [reply]
There is no options.yml file. There is a settings.yml file but it doesn't have any reference to a storage path. I´ve found the file docker-compose.yml but I'm not sure if that is the correct place to perform these adjustments...

**Reply by graciousgrey** [reply]
In the docker-compose.yml file you should be able to configure the volumes as described here:

https://docs.photoprism.app/getting-started/docker-compose/#volumes

Please note that changing values in a docker-compose.yml file always requires a restart to take effect.

**Comment by giancagentile** [comment]
Thanks!! I think its working!!
